Form-Controls/index.html

```diff
@@ -9,10 +9,35 @@
   </head>
   <body>
     <header>
-      <h1>Product Pick</h1>
+      <h1>Please choose your T-Shirt</h1>
     </header>
     <main>
       <form>
+       <p> <span style="color:red">*</span><span><label for="FullName">Name: </label><input type="text" id="FullName" name="FullName" required pattern="[A-Za-z]{3,}"></span></p>
+        <span style="color:red">*</span><label for="EmailAddress">Your Email Address: </label><input type="text" id="EmailAddress" name="EmailAddress" required pattern="[a-z0-9._%+\-]+@[a-z0-9.\-]+\.[a-z]{2,}$"></span>
```

Comment:
Please, format your code for better readability - split it to new lines, don't group many elements into one line.